### DISCUSSION TITLE
ssh on fresh install on Zero 3W

### DISCUSSION THREAD

**Discussion by bhouseski** [Joshua-Riek/ubuntu-rockchip#615] [General]
can one ssh into a fresh install?  I am having issues with video out, but it appears I cannot ssh into a fresh install of Ubuntu desktop until I finish the install process after first boot (which I cannot do since I currently do not have video out).  Does the server version allow for ssh on first boot?

**Comment by Joshua-Riek** [comment]
Yes server version allows for ssh on first boot. For desktop you need a display to go through the install wizard.

**Comment by bhouseski** [comment]
oh well, worth a try.  

onto server I go
